src/gatsby-ssr.js

```diff
@@ -1,23 +1,61 @@
-const React = require('react')
+const React = require('react');
 
 exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
-  const domain = pluginOptions.domain || 'cdn.simpleanalytics.io'
-  const scriptName = domain === 'cdn.simpleanalytics.io' ? 'hello.js' : 'app.js'
-  const scriptUrl = `https://${domain}/${scriptName}`
+  const domain = pluginOptions.domain || 'cdn.simpleanalytics.io';
+  const scriptName =
+    domain === 'cdn.simpleanalytics.io' ? 'hello.js' : 'app.js';
+  const scriptUrl = `https://${domain}/${scriptName}`;
 
   const options = {
-    key: 'plugin-simpleanalytics',
     src: scriptUrl,
-    async: true,
-    defer: true
-  }
+    async: true
+  };
 
   if (pluginOptions.metomic) {
-    options.type = 'text/x-metomic'
-    options['data-micropolicy'] = pluginOptions.metomic
+    options.type = 'text/x-metomic';
+    options.metomic = pluginOptions.metomic;
   }
 
   setHeadComponents([
-    React.createElement('script', options)
-  ])
-}
+    React.createElement('script', {
+      key: 'plugin-simpleanalytics',
+      dangerouslySetInnerHTML: {
+        __html: loadScript(domain, options)
+      }
+    })
+  ]);
+};
+
+// this script runs when the page loads, it does the following;
+// 1. Checks the doNotTrack option, and if set then it aborts and does nothing
+// 2. Creates a script tag for the Simple Analytics script
+// 3. Sets all the options properties on the script tag
+// 4. Inserts the script tag before the first other script in the document
+const loadScript = (domain, options) => {
+  return `!(function(s, i, m, p, l, e) {
+    if (
+      s.doNotTrack == "1" ||
+      m.doNotTrack == "yes" ||
+      m.doNotTrack == "1" ||
+      m.msDoNotTrack == "1" ||
+      (s.external.msTrackingProtectionEnabled && s.external.msTrackingProtectionEnabled())
```

Comment:
This is deprecated, I would suggest to remove it.

https://developer.mozilla.org/en-US/docs/Web/API/Window/external

Reply:
Depends on browser support I suppose. This line would be for <IE9 I think? Which your probably safe not supporting but please confirm :)

Reply:
I'm very happy to drop support for <IE8, if you are still using that you have serious issues and privacy shouldn't be your biggest priority. Security is more important in that case I would say.